scripts/populate_review_step.py

```diff
@@ -51,7 +63,7 @@ def _run(self):
         self.lims.put_batch(artifacts_to_upload)
 
     def add_artifact_info(self, sample):
-        rest_entities, artifacts = self.check_rest_data_and_artifacts(sample.name, 'match')
+        rest_entities, artifacts = self.check_rest_data_and_artifacts(sample.name, 'where')
```

Comment:
Now that there are no occurrences of `aggregate/...` in any instances of `self.endpoint`, the 'selector' argument is always going to be 'where'. This can be removed as an arg and `self.check_rest_data_and_artifacts` can just hard-code it.